src/scripts/main.js

```diff
@@ -1396,6 +1397,45 @@
         }
     }
 
+    /**
+     * Report the tower stage - outside, eclipse, tower floor 1-7, 9-15, 25+-31+, umbra run
+     * @param {Object <string, any>} message The message to be sent.
+     * @param {Object <string, any>} user The user state object, when the hunt was invoked (pre-hunt).
+     * @param {Object <string, any>} user_post The user state object, after the hunt.
+     * @param {Object <string, any>} hunt The journal entry corresponding to the active hunt.
+     */
+    function addValourRiftStage(message, user, user_post, hunt) {
+        const attrs = user.environment_atts;
```

Comment:
Not sure if this affects anything, but we typically use `user.quests` to parse client-side state. `user.environment_atts` is probably updated just as frequently, but for consistency's sake it might be good to stick with `user.quests.QuestRiftValour`

Reply:
Hmm, it doesn't look like `user.quests` has `active_augmentations` (at least in my case - I'm currently inside tower without any active). Might be worth making an exception then.

Reply:
yes, the quests doesn't have the active_augmentation. It does have the other attributes, but the game code uses the env_attrs, so I think it's safer to use that.